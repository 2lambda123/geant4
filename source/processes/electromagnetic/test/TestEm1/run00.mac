# $Id: run00.mac,v 1.1 1999-01-08 16:32:38 gunter Exp $
#
# Macro file for "TestEm1.cc"
# (can be run in batch, without graphic)
#
# geantino 1 GeV; 10000 evts
#
/control/verbose 2
#
/run/initialize
#
/run/verbose 2
/event/drawTracks none
#
/gun/particle geantino
/gun/energy 1 GeV
/run/beamOn 10000
