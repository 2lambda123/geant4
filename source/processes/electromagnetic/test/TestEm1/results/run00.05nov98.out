
1           USER'S DIRECTIVES TO RUN THIS JOB
            ----------------------------------


 ***** DATA CARD CONTENT      LIST                                                                            
 ***** DATA CARD CONTENT      C                                                                               
 ***** DATA CARD CONTENT      C       geantino; 10000 evts                                                    
 ***** DATA CARD CONTENT      C                                                                               
 ***** DATA CARD CONTENT      CALOR   5 (Imat) 2000. (size) 0. (field)                                        
 ***** DATA CARD CONTENT      TRIG    10000                                                                   
 ***** DATA CARD CONTENT      KINE    48 (Itype)   1.0 (Ekine)                                                
 ***** DATA CARD CONTENT      DEBUG   10  5 1000                                                              
 ***** DATA CARD CONTENT      SWIT    0 (draw) 0 (save)                                                       
 ***** DATA CARD CONTENT      CUTS    10.E-6 (cutgam)    10.E-6 (cutele)  3*10.E-3 (cutneu/had/muo)           
 ***** DATA CARD CONTENT            2*10.E-6 (bcute/m) 2*10.E-6 (dcute/m)   10.E-3 (ppcutm)                   
 ***** DATA CARD CONTENT      ABAN    0                                                                       
 ***** DATA CARD CONTENT      LOSS    1                                                                       
 ***** DATA CARD CONTENT      TIME    2=1.                                                                    
 ***** DATA CARD CONTENT      END                                                                             
 ***** DATA CARD CONTENT     END                                                                             

1************************************************************
 *                                                          *
 *    G E A N T  Version 3.2111      DATE/TIME 981105/1423  *
 *                                                          *
 *                      R U N      1                        *
 *                                                          *
 ************************************************************
 *                                                          *
 *      Data structure   Date   Time    GVERSN    ZVERSN    *
 *      --------------   ----   ----    ------    ------    *
 *                                                          *
 *           INIT       981105  1423    3.2111     3.77     *
 *                                                          *
 *           KINE       981105  1423    3.2111     3.77     *
 *                                                          *
 *           HITS       981105  1423    3.2111     3.77     *
 *                                                          *
 *           DIGI       981105  1423    3.2111     3.77     *
 *                                                          *
 *     Random number seeds:          9876        54321      *
 *     --------------------                                 *
 *                                                          *
 *----------------------------------------------------------*
 *                                                          *
 *              Standard TPAR for this run are              *
 *              ------------------------------              *
 *                                                          *
 *  CUTGAM= 10.00 keV  CUTELE= 10.00 keV  CUTNEU= 10.00 MeV *
 *  CUTHAD= 10.00 MeV  CUTMUO= 10.00 MeV                    *
 *  BCUTE = 10.00 keV  BCUTM = 10.00 keV                    *
 *  DCUTE = 10.00 keV  DCUTM = 10.00 keV  PPCUTM= 10.00 MeV *
 *  IPAIR =        1.  ICOMP =        1.  IPHOT =        1. *
 *  IPFIS =        0.  IDRAY =        1.  IANNI =        1. *
 *  IBREM =        1.  IHADR =        1.  IMUNU =        1. *
 *  IDCAY =        1.  ILOSS =        1.  IMULS =        1. *
 *  IRAYL =        0.  ILABS =        0.  ISYNC =        0. *
 *  ISTRA =        0.                                       *
 *                                                          *
 *                                                          *
 ************************************************************


0===================================================     MATERIALS      ==================================================
0MATERIAL                           A         Z     DENSITY  RADIAT L  ABSORP L NMIXT

        1 Air                     14.607     7.300      .001  .284E+05  .808E+05   2     A      Z     W
                                                                                        14.01   7.00   .700
                                                                                        16.00   8.00   .300
        2 H2 Liquid                1.010     1.000      .071  .865E+03  .790E+03   1
        3 Water                   14.320     7.215     1.000  .358E+02  .949E+02   2     A      Z     W
                                                                                         1.01   1.00   .112
                                                                                        16.00   8.00   .888
        4 Liquid Ar               39.950    18.000     1.390  .140E+02  .840E+02   1
        5 Aluminium               26.980    13.000     2.700  .890E+01  .372E+02   1
        6 Iron                    55.850    26.000     7.870  .176E+01  .171E+02   1
        7 Lead                   207.190    82.000    11.350  .560E+00  .185E+02   1
        8 Uranium                238.030    92.000    18.950  .320E+00  .120E+02   1
0===================================================   TRACKING MEDIA   ==================================================
0TMED                          MATERIAL ISVOL IFIELD  FIELDM  TMAXFD  STEMAX    DEEMAX   EPSIL   STMIN

      1 Container                  5       0     0       .00   10.00  .100E+11    .183    .001    .017
0===================================================      VOLUMES       ==================================================
0VOLUME NAME  NUMED SHAPE NPAR  PARAMETERS

      1 aBox    1   BOX     3    .100E+04  .100E+04  .100E+04

   Kine     Name          Type     Px         Py         Pz       Energy     Vorigin  Vgener
     1      GEANTINO       48     1.0000      .0000      .0000     1.0000     1.0000
 **** GTRIGI: IEVENT=   1000 IDEVT=   1000 Random Seeds =       9876       54321
 **** GTRIGI: IEVENT=   2000 IDEVT=   2000 Random Seeds =       9876       54321
 **** GTRIGI: IEVENT=   3000 IDEVT=   3000 Random Seeds =       9876       54321
 **** GTRIGI: IEVENT=   4000 IDEVT=   4000 Random Seeds =       9876       54321
 **** GTRIGI: IEVENT=   5000 IDEVT=   5000 Random Seeds =       9876       54321
 **** GTRIGI: IEVENT=   6000 IDEVT=   6000 Random Seeds =       9876       54321
 **** GTRIGI: IEVENT=   7000 IDEVT=   7000 Random Seeds =       9876       54321
 **** GTRIGI: IEVENT=   8000 IDEVT=   8000 Random Seeds =       9876       54321
 **** GTRIGI: IEVENT=   9000 IDEVT=   9000 Random Seeds =       9876       54321
 **** GTRIGI: IEVENT=  10000 IDEVT=  10000 Random Seeds =       9876       54321


 nb tracks/event neutral:    1.00  charged:     .00
 nb steps/event  neutral:    3.00  charged:     .00


 nb of process calls per event: 
        NEXT     MULS     LOSS     FIEL     DCAY     PAIR     COMP     PHOT     BREM     DRAY     ANNI     MUNU
        1.00      .00      .00      .00      .00      .00      .00      .00      .00      .00      .00      .00


1         **** NUMBER OF EVENTS PROCESSED =     10000
          **** RANDOM NUMBER GENERATOR AFTER LAST COMPLETE EVENT         9876       54321
          **** TIME TO PROCESS ONE EVENT IS =     .0002 SECONDS



Constructing Visualization Manager....
Initialising Visualization Manager....
Registering graphics systems....

***** Table : Nb of materials = 8 *****

 Material:      Air  density:  1.290 mg/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength: 285.161   m
   --->  Element: Nitrogen  N   Z =  7.0   N =  14.0   A =  14.01 g/mole  fractionMass:  70.00 %  Abundance  72.71 %
   --->  Element:   Oxygen  O   Z =  8.0   N =  16.0   A =  16.00 g/mole  fractionMass:  30.00 %  Abundance  27.29 %

 Material: H2liquid  density: 70.800 mg/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength:   8.923   m
   --->  Element: Hydrogen  H   Z =  1.0   N =   1.0   A =   1.01 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:    Water  density:  1.000  g/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength:  36.092  cm
   --->  Element: Hydrogen  H   Z =  1.0   N =   1.0   A =   1.01 g/mole  fractionMass:  11.21 %  Abundance  66.67 %
   --->  Element:   Oxygen  O   Z =  8.0   N =  16.0   A =  16.00 g/mole  fractionMass:  88.79 %  Abundance  33.33 %

 Material: liquidArgon  density:  1.390  g/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength:  14.065  cm
   --->  Element: liquidArgon      Z = 18.0   N =  40.0   A =  39.95 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material: Aluminium  density:  2.700  g/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength:   8.893  cm
   --->  Element: Aluminium      Z = 13.0   N =  27.0   A =  26.98 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:     Iron  density:  7.870  g/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength:   1.759  cm
   --->  Element:     Iron      Z = 26.0   N =  55.9   A =  55.85 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:     Lead  density: 11.350  g/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength:   5.612  mm
   --->  Element:     Lead      Z = 82.0   N = 207.2   A = 207.19 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:  Uranium  density: 18.950  g/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength:   3.861  mm
   --->  Element:  Uranium      Z = 82.0   N = 238.0   A = 238.03 g/mole  fractionMass: 100.00 %  Abundance 100.00 %



 The Box is 20   m of Aluminium
Em1PhysicsList::SetCuts:CutLength : 1 (mm)

phot:  Total cross sections from a parametrisation(L.Urban). Good description from 10 KeV to 50 MeV for all Z
        PhysicsTables from 10 keV to 50 MeV in 100 bins. 

compt:  Total cross sections from a parametrisation(L.Urban). Good description from 10 KeV to (100/Z) GeV. 
        Scattered gamma energy according Klein-Nishina.
        PhysicsTables from 10 keV to 1e+02 GeV in 100 bins. 

conv:  Total cross sections from a parametrisation(L.Urban). Good description from 1.5 MeV to 100 GeV for all Z. 
        e+e- energies according Bethe-Heitler
        PhysicsTables from 1 MeV to 1e+02 GeV in 100 bins. 

msc:   Tables of transport mean free paths.
          New model of MSC , computes the lateral 
          displacement of the particle , too.
        PhysicsTables from 1e+02  eV to 1e+02 TeV in 100 bins. 

eIoni:  delta cross sections from Moller+Bhabha. Good description from 1 KeV to 100 GeV.
        delta ray energy sampled from  differential Xsection.
        PhysicsTables from 1 keV to 1e+02 TeV in 100 bins. 

eBrem:  Total cross sections from a parametrisation(L.Urban). Good description from 1 KeV to 100 GeV.
        log scale extrapolation above 100 GeV 
        Gamma energy sampled from a parametrised formula.
        PhysicsTables from 1 keV to 1e+02 TeV in 100 bins. 

annihil:  Total cross section from Heilter formula (annihilation into 2 photons).
        gamma energies sampled according Heitler
        PhysicsTables from 10 keV to 10 TeV in 100 bins. 

msc:   Tables of transport mean free paths.
          New model of MSC , computes the lateral 
          displacement of the particle , too.
        PhysicsTables from 1e+02  eV to 1e+02 TeV in 100 bins. 

hIoni:    Knock-on electron cross sections . 
         Good description above the mean excitation energy.
         delta ray energy sampled from  differential Xsection.
        PhysicsTables from 1 keV to 1e+02 TeV in 100 bins. 

msc:   Tables of transport mean free paths.
          New model of MSC , computes the lateral 
          displacement of the particle , too.
        PhysicsTables from 1e+02  eV to 1e+02 TeV in 100 bins. 

MuIoni:  knock-on electron cross sections .
          Good description above the mean excitation energy.
         delta ray energy sampled from  differential Xsection.
      PhysicsTables from 1 keV to 1e+03 PeV in 100 bins. 

MuBrems:  cross sections from R. Kokoulin 
          Good description up to 1000 TeV.
    PhysicsTables from 1 GeV to 1e+03 PeV in 100 bins. 

MuPairProd:  cross sections from R. Kokoulin 
          Good description up to 1000 TeV.
    PhysicsTables from 1 GeV to 1e+03 PeV in 50 bins. 
# $Id: run00.05nov98.out,v 1.1 1999-01-08 16:32:44 gunter Exp $
#
# Macro file for "TestEm1.cc"
# (can be run in batch, without graphic)
#
# geantino 1 GeV; 10000 evts
#
#
/run/initialize
#
/run/verbose 2
/event/drawTracks none
#
/gun/particle geantino
/gun/energy 1 GeV
/run/beamOn 10000
### Run 0 start.
Start closing geometry.
Start Run processing.
Run terminated.
Run Summary
  Number of events processed : 10000
  User=5.7s Real=39s Sys=14s

 nb tracks/event   neutral:    1.00   charged:    0.00
 nb  steps/event   neutral:    1.00   charged:    0.00

 nb of process calls per event: 
        geom
        1.00
